### PR TITLE
Changed documentation about xml parser types [#4121]

### PR DESCRIPTION
**Pull request by coquisx**
Changed documentation about XML parser types, adapting it to DDSXTypes standard

### CODE REVIEW

File: docs/xmlprofiles.rst [hunk 1]
```diff
@@ -234,25 +234,22 @@ Member types
...
+| ``<char>``               | ``<uint32>``             | ``<wstring>``            |
 +--------------------------+--------------------------+--------------------------+
-| ``<wchar>``              | ``<unsignedlonglong>``   | ``<boundedString>``      |
+| ``<wchar>``              | ``<uint64>``             |                          |
 +--------------------------+--------------------------+--------------------------+
-| ``<short>``              | ``<float>``              | ``<boundedWString>``     |
+| ``<int16>``              | ``<float32>``            |                          |
 +--------------------------+--------------------------+--------------------------+
-| ``<long>``               | ``<double>``             |                          |
+| ``<int32>``              | ``<float64>``            |                          |
```
Comment (LuisGP) [inline docs/xmlprofiles.rst]: This section must be updated to reflect that now these keywords are `type` attributes instead tags. And remember to remove the tag symbols (`<` and `>`).

Comment (coquisx) [reply docs/xmlprofiles.rst]: Done

**Comment by LuisGP** [comment]
Updated. Please, check first PR [#369](https://github.com/eProsima/Fast-RTPS/pull/369) of Fast-RTPS

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/16/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/18/)

**Pull request merged**